src/dispatcher/index.ts

```diff
@@ -192,37 +191,61 @@ const createDispatcher = () => {
           }
         : // istanbul ignore next
           undefined,
-    _toString: () => {
+    _static: () => {
       //  Will process the two arrays, taking the first title in the array and returning <title>{string}</title>
       //  Then do a similar for the meta's. (will also need to add links, and add a linkQueue). Note that both queues
       //  will need a reset to prevent memory leaks.
       const visited = new Set();
-      const title = applyTitleTemplate(titleQueue[0], titleTemplateQueue[0]);
+      const title = applyTitleTemplate(
+        titleQueue[titleQueue.length - 1],
+        titleTemplateQueue[titleTemplateQueue.length - 1]
+      );
+      metaQueue.reverse();
+
+      // @ts-ignore
+      const metas = [...metaQueue].filter((meta) => {
+        if (!visited.has(meta.charset ? meta.keyword : meta[meta.keyword])) {
+          visited.add(meta.charset ? meta.keyword : meta[meta.keyword]);
+          return true;
+        }
+      });
 
       const stringified = `
```

Comment:
Contemplating on removing the `stringified`

Reply:
I agree. :+1: